src/schema.json

```diff
@@ -0,0 +1,283 @@
+{
```

Comment:
We need to add the command line options (including the short option) to the schema as they can be expressed in the dojorc also.